metrics.md

```diff
@@ -236,10 +238,12 @@ $$ BSS = 1 - \frac{\text{BS}_f}{\text{BS}_{\text{ref}}} $$
 
 where $$ \text{BS}_f $$ is the BS of the forecast of interest, and $$ \text{BS}_{\text{ref}} $$ is the BS of the reference forecast. BSS greater than zero indicates the forecast performed better than the reference and vice versa for BSS less than zero, while BSS equal to zero indicates the forecast is no better (or worse) than the reference.
 
-When the probability forecast takes on a finite number of values (e.g. 0.0, 0.1, ..., 0.9, 1.0), the BS can be decomposed into a sum of three metrics that give additional insight into a probability forecast:
+When the probability forecast takes on a finite number of values (e.g. 0.0, 0.1, ..., 0.9, 1.0), the BS can be decomposed into a sum of three metrics that give additional insight into a probability forecast ([Murphy73](#ref-murphy73)):
```

Comment:
BSS rather than BS here?  (actively resisting the obvious jokes about the acronyms)

Reply:
Actually, it should be Brier Score (BS), not Brier Skill Score (BSS). But this section should probably be moved elsewhere (e.g. into the BS section).

Reply:
I moved the above text into the end of the Brier Score section, and moved the Brier Skill Score to after the reliability, resolution and uncertainty definitions.